ExcelMakerConsole/ExcelFileMaker.cs

```diff
@@ -209,45 +208,70 @@ internal void exportExcelFile()
                 
                 foreach (Sample sample in group.samples)
                 {
-                    double[] timeData = sample.column1.data;
-                    double[] stressData = sample.column2.data;
-                    double[] strainData = sample.column3.data;
-                    double[] strainRateData = sample.column4.data;
+                    double[] timeData = sample.columns[0].data;
+                    double[] stressData = sample.columns[1].data;
+                    double[] strainData = sample.columns[2].data;
+                    double[] strainRateData = sample.columns[3].data;
+                    double[] frontFaceData = new double[0];
+                    double[] backFaceData = new double[0];
+                    if (sample.has_face_force())
+                    {
+                        frontFaceData = sample.columns[4].data;
+                        backFaceData = sample.columns[5].data;
+                    }

```

Comment:
Looks like we can move all this face force code (lines 215-221) to inside the `sample.has_face_force()` conditional on line 254. Minimize the scope of variables